tools/snp.sh

```diff
@@ -700,7 +688,7 @@ build_and_install_amdsev() {
   if [ "$SVSM" = true ]; then
```

Comment:
Logic backward. Please Fix.

Reply:
No. You can't skip this for the sake of speeding up the merging of your PR. Fixing the logic for this does nothing to break any level of the build process. It literally is centered around a conditional change. If you don't do clean up, and you were to build the SVSM then build the standard image, it will break the image as it doesn't clean up the SVSM from the previous build. You need to address it and fix it.